.github/workflows/publish.yml

```diff
@@ -4,14 +4,10 @@ on:
   push:
     branches: # Snapshots
       - main
-    tags: ["*"] # Releases
```

Comment:
Not 100% sure but I think double quotes may be seen as normal string? Only when using singlel quotes a string is escaped?